openedx/core/djangoapps/user_api/accounts/tests/test_api.py

```diff
@@ -32,7 +33,7 @@ def mock_render_to_string(template_name, context):
 
 
 @unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Account APIs are only supported in LMS')
-class TestAccountApi(TestCase):
+class TestAccountApi(EventTestMixin, TestCase):
```

Comment:
Nit: could probably use UserSettingsEventTestMixin now instead of EventTestMixin, which would set the tracker name.